doc/15-troubleshooting.md

```diff
@@ -176,6 +176,60 @@ C:\> cd C:\ProgramData\icinga2\var\log\icinga2
 C:\ProgramData\icinga2\var\log\icinga2> Get-Content .\debug.log -tail 10 -wait
 ```
 
+### Enable/Disable Debug Output on the fly <a id="troubleshooting-enable-disable-debug-output-api"></a>
+
+Every feature is just an Icinga config object similar to Host and Service, e.g.
+the `debuglog` feature is a [FileLogger](09-object-types.md#objecttype-filelogger).
+Those can also be [managed via API](12-icinga2-api.md#icinga2-api-config-objects)
+at runtime. This is a good alternative to `icinga2 feature enable debuglog`:
+
+* Object creation/deletion via API happens immediately and requires no restart
+* Hence, the debug log is enabled exactly as long as desired
+
+However, in case of [a HA zone](06-distributed-monitoring.md#distributed-monitoring-scenarios-ha-master-agents)
+the feature gets enabled/disabled on both nodes once they're connected.
```

Comment:
# ⚠️ Thing to consider

* #10022

Reply:
What did happen to this consideration? I mean it's clearly a bug, but I didn't get why posted such a prominent warning here and doing nothing about it.

Reply:
> why posted such a prominent warning here

This PR (necessarily) "sells" the user a shiny HA feature which has a bug which may negatively affect what the user's doing inspired by this PR. Hence, you as reviewers may reconsider the PR as whole...

> and doing nothing about it

... because, should we really make a disclaimer in the docs like, HA may be not reliable?

Reply:
> ... because, should we really make a disclaimer in the docs like, HA may be not reliable?

No, I wouldn't do it either, as the bug literally impacts every Icinga 2 object. I just didn't get the purpose of this comment with a prominent and eye catching warning sign.